en/docs/install-and-setup/setup/security/user-account-management-for-dev-portal.md

```diff
@@ -0,0 +1,178 @@
+# User Account Management for the Developer Portal
+
+The following sections explain how to manage Developer Portal related user accounts.
+
+## Password recovery
+
+The password recovery feature does not work out-of-the-box because an email server needs to be configured to be able to send the password recovery email. 
+
+Follow the steps below to enable the password recovery feature for the Developer Portal:
+
+1. Shut down the server if it is running.
+
+2. Open the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+3. Add the following properties to configure the a e-mail server. 
+
+     For more information, see [Configuring the Email Sending Module](https://is.docs.wso2.com/en/5.10.0/setup/configuring-email-sending/).
+
+    ``` java
+    [output_adapter.email]
+    from_address = "wso2am@gmail.com"
+    username = "wso2amtest"
+    password = "Wso2@am70"
+    hostname = "smtp.gmail.com"
+    port = "587"
+    enable_start_tls = true
+    enable_authentication = true
+    ```
+
+4. Re-start the server.
+
+## Account locking
```

Comment:
this also is applicable only for dev portal?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3